src/DDTrace/Integrations/Lumen/LumenSandboxedIntegration.php

```diff
@@ -63,14 +63,17 @@ function (SpanData $span, $args) use ($rootSpan, $integration, $appName) {
             }
         );
 
+        // convert to non-tracing API
+        $hook = 'posthook';
         // Extracting resource name as in legacy integration
         \DDTrace\trace_method(
             'Laravel\Lumen\Application',
             'handleFoundRoute',
             [
-                'prehook' => function (SpanData $span, $args) use ($rootSpan) {
+                $hook => function (SpanData $span, $args) use ($rootSpan, $appName) {
+                    $span->service = $appName;
```

Comment:
I just realized we need to also set `$span->type = 'web';` to be consistent with the other instrumented bits from the Laravel integration.

Reply:
Thanks; set this in most recent version.